hw3/src/CountryList.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import styles from './CountryList.module.css';
+
+const CountryList = () => {
+    const [countries, setCountries] = useState([]);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        axios.get('https://cs464p564-frontend-api.vercel.app/api/countries')
+            .then(response => {
+                setCountries(response.data);
+            })
+            .catch(error => {
+                setError('Error fetching country data.');
+            });
+    }, []);
+
+    if (error) {
+        return <div>{error}</div>;
+    }
+
+    return (
+        <div className={styles.countryListContainer}>
```

Comment:
Your country cards remain in a singular column regardless of how big the screen size is. Maybe consider changing it to fit the screen size so there isn't as much scrolling required when widescreen.

Reply:
I have now wrapped around the country cards.